src/psm/src/ir_solver.cpp

```diff
@@ -807,7 +807,7 @@ void IRSolver::buildCondMatrixAndVoltages(
       J[node_idx] = find_node->second;
     }
 
-    Connection::Conductance node_cond = 0.0;
+    long double node_cond = 0.0;
```

Comment:
Maybe we should add another using declartation called `Connection::ConductanceHighPrecision`

Reply:
long double may have performance implications (using x87 and not SSE).  Is this strictly necessary?

Reply:
I think I found a seconds stop that could avoid this... so I converted to draft until I can verify.